README.md

```diff
@@ -1,24 +1,47 @@
-# README
+#Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
```

Comment:
- [ ] Please try to deploy the application as required in the project requirements.